packages/client/src/views/base-view-module.ts

```diff
@@ -0,0 +1,98 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { ContainerModule, interfaces } from 'inversify';
+import {
+    CircularNode,
+    CircularNodeView,
+    configureModelElement,
+    DiamondNode,
+    DiamondNodeView,
+    ExpandButtonView,
+    ForeignObjectElement,
+    ForeignObjectView,
+    HtmlRoot,
+    HtmlRootView,
+    PreRenderedElement,
+    PreRenderedView,
+    RectangularNode,
+    RectangularNodeView,
+    SButton,
+    SCompartment,
+    SCompartmentView,
+    SEdge,
+    SGraphView,
+    ShapedPreRenderedElement,
+    SIssueMarker,
+    SLabel,
+    SLabelView,
+    SNode,
+    SPort,
+    SRoutingHandle,
+    SRoutingHandleView,
+    SvgViewportView,
+    ViewportRootElement
+} from 'sprotty';
+
+import { GLSPGraph } from '../lib/model';
+import { DefaultTypes } from './default-types';
+import { GEdgeView } from './glsp-edge-view';
+import { GIssueMarkerView } from './issue-marker-view';
+import { RoundedCornerNodeView } from './rounded-corner-view';
+
+interface ContainerContext {
+    bind: interfaces.Bind;
```

Comment:
The `configureModelElement` from Sprotty which we call only needs those two. Since this is not an exported interface anyway, I believe we can leave it as is, what do you think?